backends/tests/CMakeLists.txt

```diff
@@ -11,7 +11,7 @@ else()
   # We need thread support for gtest
   find_package(Threads REQUIRED)
 
-  set(CMAKE_CTEST_COMMAND ctest -V) 
+  set(CMAKE_CTEST_COMMAND ctest -V)
```

Comment:
@diptorupd How do you start tests for C++ code? Could you write please your method in README or other place?

Reply:
Run `ninja check` or `make check`.

Reply:
`make check` or `ninja check`